ROLES.md

```diff
@@ -325,55 +243,50 @@ be the tech lead in a smaller working group, but they are distinct roles.
 
   - Report. Report current status to the TOC meeting every 6 weeks.
 
-- Holistic responsibility for their working group's [feature
-  tracks](./mechanics/FEATURE-TRACKS.md): tracking, health, and execution.
-
-- Perform issue triage on GitHub.
+- Expected to work to holistically maintain the health of the project through:
 
-- Apply/remove/create/delete GitHub labels and milestones.
+  - Reviewing PRs
 
-- Write access to repo (assign issues/PRs, add/remove labels and milestones,
-  edit project, issues, and PRs, edit wiki, create/delete labels and milestones).
+  - Fixing bugs
 
-- Expected to work to holistically maintain the health of the working group through:
+  - Identifying needed enhancements / areas for improvement / etc.
 
-  - Being a good role model
+  - Execute pay-down of technical debt
```

Comment:
I'd probably say "manage" -- in some cases, technical debt might not be the largest issue facing a working group, so I'd be cautious about making it a bright-line requirement.

Reply:
This was carried over from the original document and I considered removing it but didn't - happy to strike it.